index.php

```diff
@@ -120,29 +131,15 @@ function mysort($a, $b)
                                 <!-- Tree Death Count -->
                                 <td>' . $db[substr($file, 0, -5) . '.sav'][6] . '</td>
 
-                                <!-- Locomotives Owned -->
+                                <!-- Locos -->
                                 <td>' . $db[substr($file, 0, -5) . '.sav'][2] . '</td>
 
-                                <!-- Rolling Stock Owned -->
+                                <!-- Carts -->
                                 <td>' . $db[substr($file, 0, -5) . '.sav'][3] . '</td>
 
                                 <!-- Max Slope -->
-                                <td>' . round($db[substr($file, 0, -5) . '.sav'][4]) . '%</td>
-                                <!-- Shared Link -->';
-
-                            // Checks public save folder to see if we can provide a link
-                            $saveCheck = SHELL_ROOT . 'saves/public/' . substr($file, 5, -5) . '.sav';
-                            if (file_exists($saveCheck)) {
-                                $upTime = filemtime($saveCheck);
-                                $timeCheck = time() - $upTime;
-                                if ($timeCheck < 172800) {
-                                    echo '<td><a href="' . WWW_ROOT . 'saves/public/' . substr($file, 5, -5) . '.sav">Link</a></td>';
```

Comment:
uh... here we had this magic number... ;-)

Reply:
Sorry not sure what you mean by magic number?